.github/container/Dockerfile.t5x.arm64

```diff
@@ -0,0 +1,226 @@
+RUN echo "grain @ file://$(ls /opt/grain*.whl)" >> /opt/pip-tools.d/manifest.pax
+
+
+### --- TO BE REFACTORED? (everything below is as in @nouiz's original PR) ---
+
+# force a recent version to have latest protobuf dep
+# Install now as some build need them.
+RUN pip install "tensorflow_datasets==4.9.2"
+RUN pip install "tensorflow==2.13.0"
+RUN pip install "chex==0.1.7"
+RUN <<"EOF" bash -ex
+cd /opt
+git clone http://github.com/tensorflow/text.git
+cd text
+git checkout v2.13.0
+EOF
+
+RUN <<"EOF" bash -ex
+cd /opt/text
+
+wget https://github.com/bazelbuild/bazelisk/releases/download/v1.17.0/bazelisk-linux-arm64 -O /usr/bin/bazel ;
+chmod a+x /usr/bin/bazel
+
+./oss_scripts/run_build.sh
+find * | grep '.whl$'
+EOF
+RUN pip install /opt/text/tensorflow_text-*.whl
+#RUN echo "-e file:///opt/text/tensorflow_text-*.whl"  >> /opt/pip-tools.d/manifest.t5x
+RUN echo "tensorflow-text @  file://$(ls /opt/text/tensorflow_text-*.whl)" >> /opt/pip-tools.d/manifest.t5x
+
+# Install T5 now, Pip will build the wheel from source, it needs Rust.
+RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs > /tmp/rustup.sh && \
+    echo "be3535b3033ff5e0ecc4d589a35d3656f681332f860c5fd6684859970165ddcc /tmp/rustup.sh" | sha256sum --check && \
+    bash /tmp/rustup.sh -y && \
+    export PATH=$PATH:/root/.cargo/bin && \
+    pip install t5 && \
+    rm -Rf /root/.cargo /root/.rustup && \
+    mv /root/.profile /root/.profile.save && \
+    grep -v cargo /root/.profile.save > /root/.profile && \
+    rm /root/.profile.save && \
+    mv /root/.bashrc /root/.bashrc.save && \
+    grep -v cargo /root/.bashrc.save > /root/.bashrc && \
+    rm /root/.bashrc.save && \
+        rm -Rf /root/.cache /tmp/*
+
+# NOTE(gspschmid): Broken? Should've been "echo t5"?
+# RUN "t5 >> /opt/pip-tools.d/manifest.t5x"
+
+RUN <<"EOF" bash -ex
+get-source.sh -f ${REPO_T5X} -r ${REF_T5X} -d ${SRC_PATH_T5X}
+echo "-e file://${SRC_PATH_T5X}[gpu]" >> /opt/pip-tools.d/manifest.t5x
+
+# remove head-of-tree specs from select dependencies
+pushd ${SRC_PATH_T5X}
+sed -i "s| @ git+https://github.com/google/flax#egg=flax||g" setup.py
+# for ARM64 build
+sed -i "s/tensorflow/#tensorflow/" setup.py
+sed -i "s/t5=/#t5=/" setup.py
+sed -i "s/^jax/#jax/" setup.py
+
+sed -i "s/f'jax/#f'jax/" setup.py
+sed -i "s/'tpu/#'tpu/" setup.py
+
+sed -i 's/protobuf/#protobuf/' setup.py
+sed -i 's/numpy/#numpy/' setup.py
+
+if git diff --quiet; then
+    echo "URL specs no longer present in select dependencies of t5x"
+    exit 1
+else
+    git commit -a -m "remove URL specs from select dependencies of t5x"
+fi
+popd
+EOF
+
+ADD test-t5x.sh /usr/local/bin
+
+###############################################################################
+## Install accumulated packages from the base image and the previous stage
+###############################################################################
+
+FROM mealkit as final
+
+RUN pip uninstall -y tensorflow-text
```

Comment:
This won't be needed if we build it in a separate stage and only copy the wheel into the mealkit.